acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/lotus/FilecoinKey.java

```diff
@@ -12,70 +12,43 @@
  */
 package tech.pegasys.eth2signer.dsl.lotus;
 
-import tech.pegasys.teku.bls.BLSKeyPair;
-import tech.pegasys.teku.bls.BLSSecretKey;
+import static tech.pegasys.eth2signer.dsl.lotus.FilecoinKeyType.BLS;
 
 import java.util.Objects;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import org.apache.tuweni.bytes.Bytes;
-import org.web3j.crypto.ECKeyPair;
-import org.web3j.utils.Numeric;
 
 public class FilecoinKey {
   private final FilecoinKeyType type;
-  private final String privateKey;
-
-  private String privateKeyHex;
-  private String publicKey;
+  private final Bytes privateKey;
+  // the hex version is to be stored in Signers configuration file
+  private final String privateKeyHex;
 
   @JsonCreator
   public FilecoinKey(
```

Comment:
q: is this response from a WalletExtract?

Reply:
yes...this conforms to response from WalletExtract call (i.e. Type and PrivateKey).